audit/projects/k8s-infra-prow-build/services/container/clusters.txt

```diff
@@ -1 +1 @@
-prow-build	us-central1	us-central1-c;us-central1-f;us-central1-b	67	RUNNING
+prow-build	us-central1	us-central1-c;us-central1-f;us-central1-b	72	RUNNING
```

Comment:
nit: we should have audit dump a format that is less noisy for clusters, given that this cluster auto-scales